Makefile

```diff
@@ -59,14 +59,14 @@ test-stack-command-oldest:
 	./scripts/test-stack-command.sh 7.14.2
 
 test-stack-command-7x:
-	./scripts/test-stack-command.sh 7.17.18
+	./scripts/test-stack-command.sh 7.17.19
```

Comment:
Not updating to 7.17.20 because of https://github.com/elastic/fleet-server/issues/3435.